packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1068,4 +1069,52 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'apilintOpenAPIParameterFieldIsDefinedWithinPathTemplate',
+    function: (element: Element) => {
+      if (element.element === 'parameter') {
+        const parameterName = toValue((element as ObjectElement).get('name'));
+        const parameterLocation = toValue((element as ObjectElement).get('in'));
+        const isInPathItemElement =
+          isArrayElement(element.parent) &&
+          includesClasses(['path-item-parameters'], element.parent);
+        const isChildOfOperationElement = element.parent.parent.parent.element === 'operation';
```

Comment:
Let's not deal with `Operation.parameters` at this point of time. It will just unnecessary complicate the implementation.

Reply:
Ok. I'll remove the code for that.